src/NovelRT.Interop/PluginManagement/NrtIResourceManagementPluginProvider.cpp

```diff
@@ -41,7 +42,8 @@ NrtResult Nrt_IResourceManagementPluginProvider_GetResourceLoader(NrtIResourceMa
         return NRT_FAILURE_NULL_ARGUMENT_PROVIDED;
     }
 
-    auto&& resourceLoader = reinterpret_cast<IResourceManagementPluginProvider*>(plugin)->GetResourceLoader();
+    const bool initBool = static_cast<bool>(initAssets);
```

Comment:
is this how we handle `NrtBool` in other locations?

Reply:
Looks like there's no cast at all in other places in the code. Initially thought that a cast would be necessary to avoid some "narrowing cast is implementation-dependent" warnings but apparently that cast is defined in the standard. I'll remove that.